### PR TITLE
Order tests

### PR DESCRIPTION
**Pull request by xzzz3**

### CODE REVIEW

**Comment by xzzz3** [comment]
Removed one test case as it wasn't passing on CI but passed locally, unsure of the reason
image

File: docs/UserGuide.md [hunk 1]
```diff
@@ -352,14 +352,16 @@ Examples:
 * `editorder 1 p/87264512 d/kimchi, fried rice`
 * `editorder 3 p/87654321`
 
-### Search for an Order by Phone Number: `findorder`
 
-Search for an order with the phone number provided in the list of orders.
+### Search for Orders by Phone Number: `findorder`
```
Comment (jaegarpoon) [inline docs/UserGuide.md:356]: Maybe can change this to `find` instead!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -352,14 +352,16 @@ Examples:
...
 
-Format: `find p/PHONE`
+Search for orders with the customer phone number(s) provided in the list of orders.
+
+Format: `find p/PHONE_SEPARATED_BY_SPACE...`
```
Comment (jaegarpoon) [inline docs/UserGuide.md:360]: or `findorder`

**Review by jaegarpoon — APPROVED** [review]
Looks good! Just some small issues with UG :)

File: docs/UserGuide.md (hunk 2 above)
Comment (xzzz3) [reply docs/UserGuide.md:360]: right! will change to findorder. i think find is used by customer?? or another class

**Review by zxgoh — APPROVED** [review]
LGTM

**Pull request merged**